multisig/simulations/ethereum/vipConverter/index.ts

```diff
@@ -0,0 +1,306 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import { expect } from "chai";
+import { BigNumber, Contract, Signer } from "ethers";
+import { parseUnits } from "ethers/lib/utils";
+import { ethers } from "hardhat";
+
+import { initMainnetUser } from "../../../../src/utils";
+import { forking, pretendExecutingVip } from "../../../../src/vip-framework";
+import {
+  CONVERTER_NETWORK,
+  GUARDIAN,
+  USDC_PRIME_CONVERTER,
+  USDT_PRIME_CONVERTER,
+  XVS_VAULT_CONVERTER,
+  converters,
+} from "../../../proposals/ethereum/vip-converter/Addresses";
+import vipConverter, {
+  COMPTROLLER_CORE,
+  COMPTROLLER_LST,
+  PLP,
+  PRIME,
+  PROTOCOL_SHARE_RESERVE_PROXY,
+  SINGLE_TOKEN_CONVERTER_BEACON,
+  USDC,
+  USDT,
+  VTREASURY,
+  VUSDC_CORE,
+  VUSDT_CORE,
+  VWBTC_CORE,
+  VWETH_LST,
+  WBTC,
+  WETH,
+  XVS_VAULT_TREASURY,
+} from "../../../proposals/ethereum/vip-converter/vipConverter";
+import CONVERTER_NETWORK_ABI from "./abi/ConverterNetwork.json";
+import ERC20_ABI from "./abi/ERC20.json";
+import COMPTROLLER_ABI from "./abi/ILComptroller.json";
+import PRIME_ABI from "./abi/Prime.json";
+import PLP_ABI from "./abi/PrimeLiquidityProvider.json";
+import PROTOCOL_SHARE_RESERVE_ABI from "./abi/ProtocolShareReserve.json";
+import SINGLE_TOKEN_CONVERTER_ABI from "./abi/SingleTokenConverter.json";
+import SINGLE_TOKEN_CONVERTER_BEACON_ABI from "./abi/SingleTokenConverterBeacon.json";
+import XVS_VAULT_CONVERTER_ABI from "./abi/XVSVaultTreasury.json";
+import XVS_VAULT_TREASURY_ABI from "./abi/XVSVaultTreasury.json";
+
+const XVS = "0xd3CC9d8f3689B83c91b7B59cAB4946B063EB894A";
+const XVS_VAULT = "0xA0882C2D5DF29233A092d2887A258C2b90e9b994";
+const USDT_HOLDER = "0xb23360CCDd9Ed1b15D45E5d3824Bb409C8D7c460";
+const USDC_HOLDER = "0x974CaA59e49682CdA0AD2bbe82983419A2ECC400";
+
+forking(20120060, () => {
+  const provider = ethers.provider;
+
+  describe("Post-VIP behavior", () => {
```

Comment:
I would add check for new implementation.

Reply:
Done https://github.com/VenusProtocol/vips/pull/303/commits/52e3de55ff7f4d256ddd6c501e2ab567733ba8cd